tools/langs/manuscript.py

```diff
@@ -156,8 +169,15 @@
 GetId "(element) {
         return element._id;
 }"
-SetId "(element, value) {
-        element._id = value;
+SetId "(element, newId) {
+    UnregisterId(element._id);
+    element._id = value;
```

Comment:
Same problem here as in the sibmei repo -- I think `value` is supposed to be `newId`?